data/pigui/modules/station-view/07-police.lua

```diff
@@ -1,17 +1,154 @@
 -- Copyright © 2008-2020 Pioneer Developers. See AUTHORS.txt for details
 -- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
 
+local Game = import "Game"
+local Rand = import "Rand"
 local ui = import 'pigui/pigui.lua'
 local StationView = import 'pigui/views/station-view'
 local Lang = import 'Lang'
-
+local Legal = import "Legal"
+local utils = import "utils"
+local InfoFace = import 'ui/PiguiFace'
+local Format = import "Format"
+local Character = import "Character"
 local l = Lang.GetResource("ui-core")
 
+local ModalWindow = import 'pigui/libs/modal-win.lua'
+
+local rescaleVector = ui.rescaleUI(Vector2(1, 1), Vector2(1600, 900), true)
+
+local pionillium = ui.fonts.pionillium
+local orbiteer = ui.fonts.orbiteer
+local colors = ui.theme.colors
+local face = nil
+local stationSeed = false
+
+local popup = ModalWindow.New('policePopup', function(self)
+	ui.text(l.YOU_NOT_ENOUGH_MONEY)
+	ui.dummy(Vector2((ui.getContentRegion().x - 100*rescaleVector.x) / 2, 0))
+	ui.sameLine()
+	if ui.button(l.OK, Vector2(100*rescaleVector.x, 0)) then
+		self:close()
+	end
+end)
+
+local gray = Color(100, 100, 100)
+
+local widgetSizes = ui.rescaleUI({
+	itemSpacing = Vector2(4, 9),
+	windowPadding = Vector2(14, 14),
+	faceSize = Vector2(586,565),
+}, Vector2(1600, 900))
+
+
+local function payfine(fine)
+	if Game.player:GetMoney() < fine then
+		popup:open()
+		return
+	end
+
+	Game.player:AddMoney(-fine)
+	Game.player:ClearCrimeFine()
+end
+
+
+local function crime_record()
+	local past_crimes, stump = Game.player:GetCrimeRecord()
+	if #utils.build_array(pairs(past_crimes)) > 0 then
+		ui.withFont(orbiteer.xlarge.name, orbiteer.xlarge.size,
+					function() ui.textColored(gray, l.CRIMINAL_RECORD) end)
+		ui.withFont(pionillium.medlarge.name, pionillium.medlarge.size,
+					function ()
+						for k,v in pairs(past_crimes) do
+							ui.textColored(gray, v.count.."\t"..Legal.CrimeType[k].name)
+						end
+					end)
+	end
+end
+
+
+local function outstanding_fines()
+	local crimes, fine = Game.player:GetCrimeOutstanding()
+	if #utils.build_array(pairs(crimes)) > 0 then
+
+		-- headline
+		ui.withFont(orbiteer.xlarge.name, orbiteer.xlarge.size,
+					function() ui.text(l.OUTSTANDING_FINES) end)
+		-- wrap list in medlarge font
+		ui.withFont(pionillium.medlarge.name, pionillium.medlarge.size,
+					function()
+						for k,v in pairs(crimes) do
+							ui.text(v.count.."\t"..Legal.CrimeType[k].name)
```

Comment:
```suggestion
							ui.text(v.count)
							-- TODO: scale this spacing value with ui.rescaleUI
							ui.sameLine(40 - ui.calcTextSize(v.count))
							ui.text(Legal.CrimeType[k].name)
```

Reply:
Resolved as discussed below. Went for the fixed position, increased it from 50 to 55. 

If a player has +999 offences, me thinks we should aggregate illegal discharge offences differently. "I can't imagine anyone needing more than 999 crimes (64 kb)"